src/components/Title/index.js

```diff
@@ -4,21 +4,32 @@ import classnames from "classnames";
 
 import styles from "./index.module.css";
 
+const THEMES_VALUES = [
+  styles.redTheme,
+  styles.neonCarrotTheme,
+  styles.greyishBlueTheme,
+];
+
+export const THEMES = {
+  RED: THEMES_VALUES[0],
+  NEON_CARROT: THEMES_VALUES[1],
+  GREISH_BLUE: THEMES_VALUES[2],
+};
```

Comment:
I think we should apply the same "pattern" to keep consistency across components.
I'm ok in changing the title and bigText component back to the 1st implementation. ✌️

Reply:
Sh** I thought we agreed on exporting everything. Can we do that now?

Reply:
I'm ok with that, but what is the advantage?

Reply:
Justo started with it first, so I'm just going with the flow.

In my case, I can figure out the supported themes for each component with my text editor without actually looking at the source code of the components.

Reply:
@joaojusto , is that the purpose?

Reply:
The purpose is to avoid writing random strings as props, it's easier to find what are the options for a determined prop, you just need to follow the imports

Reply:
It also helps for those of use who have autocomplete plugins, as it is able to figure out the possible values.

Reply:
Ok 👍